crates/pbind/Cargo.toml

```diff
@@ -0,0 +1,8 @@
+[package]
+name = "pbind"
```

Comment:
hm, since this is a low-level binding to the C headers, should this perhaps be `pbind-sys`?